docs/Using-TensorFlow-Sharp-in-Unity.md

```diff
@@ -72,8 +72,9 @@ If you followed these instructions well, the agents in your environment that use
 
 # iOS additional instructions for building
 
-* Once you build for iOS in the editor, Xcode will launch.
-* In **General** > **Linked Frameworks and Libraries**:
+* Once you build the project for iOS in the editor, open the .xcodeproj file within the project folder using Xcode.
+* Set up your ios account following the [iOS Account setup page](https://docs.unity3d.com/Manual/iphone-accountsetup.html). 
+* In `General` -> `Linked Frameworks and Libraries`:
   * Add a framework called `Accelerate.framework`
```

Comment:
Maybe mark this as "If the framework `Accelerate.framework` is not added already, add it. " since you managed to add Accelerate by default.

Reply:
Yes I will update the unitypackage on s3 with the option checked, then change this line.